src/Symfony/Component/Security/Http/HttpUtils.php

```diff
@@ -144,14 +141,13 @@ public function checkRequestPath(Request $request, $path)
     /**
      * Generates a URI, based on the given path or absolute URL.
      *
-     * @param Request $request A Request instance
-     * @param string  $path    A path (an absolute path (/foo), an absolute URL (http://...), or a route name (foo))
+     * @param string $path A path (an absolute path (/foo), an absolute URL (http://...), or a route name (foo))
      *
      * @return string An absolute URL
      *
      * @throws \LogicException
      */
-    public function generateUri($request, $path)
+    public function generateUri(Request $request, $path)
```

Comment:
I ported the patch in this PR on 4.4 in #32974, but this file must be updated on master only, so it's not part of that other PR.